src/main/java/com/sap/oss/phosphor/fosstars/tool/format/CommonFormatter.java

```diff
@@ -261,20 +261,17 @@ private static void add(Feature<?> feature, String caption) {
     FEATURE_TO_NAME.put(feature, caption);
   }
 
-  /**
-   * A formatter for doubles.
-   */
-  private static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("#.#");
-
   static {
-    DECIMAL_FORMAT.setMinimumFractionDigits(1);
-    DECIMAL_FORMAT.setMaximumFractionDigits(2);
   }
```

Comment:
May I know why this was deleted and moved to a `constructor()`? :), we do not use this setting anywhere else.